app/build.gradle

```diff
@@ -42,12 +49,14 @@ dependencies {
     // However, that is a different topic.
     def daggerVersion = '2.14'
     def butterKnifeVersion = '8.8.0'
+    def kotlinVersion = '1.2.0'
 
-    annotationProcessor "com.google.dagger:dagger-compiler:$daggerVersion"
-    annotationProcessor "com.google.dagger:dagger-android-processor:$daggerVersion"
-    annotationProcessor "com.jakewharton:butterknife-compiler:$butterKnifeVersion"
+    kapt "com.google.dagger:dagger-compiler:$daggerVersion"
+    kapt "com.google.dagger:dagger-android-processor:$daggerVersion"
+    kapt "com.jakewharton:butterknife-compiler:$butterKnifeVersion"
 
     compile "com.google.dagger:dagger:$daggerVersion"
     compile "com.google.dagger:dagger-android:$daggerVersion"
     compile "com.jakewharton:butterknife:$butterKnifeVersion"
+    compile "org.jetbrains.kotlin:kotlin-stdlib-jre7:$kotlinVersion"
```

Comment:
Include the Kotlin standard library for JRE 7 since this project still targets and compiles against Java 7;

```java
compileOptions {
      sourceCompatibility JavaVersion.VERSION_1_7
      targetCompatibility JavaVersion.VERSION_1_7
}
```

On a side note, Android Studio 3.x adds native support for some of Java 8's new features like Lambdas (no need for Retrolambda). Therefore, upgrading to Android Studio 3.x will make targeting Java 8 much easier.

Reply:
`kotlin-stdlib-jre7` should instead be `kotlin-stdlib-jdk7`. This is fixed in https://github.com/vestrel00/android-dagger-butterknife-mvp/pull/72/files#r173839467